components/cli/cli/cli.go

```diff
@@ -33,6 +33,7 @@ import (
 	"cellery.io/cellery/components/cli/pkg/kubernetes"
 	"cellery.io/cellery/components/cli/pkg/registry"
 	"cellery.io/cellery/components/cli/pkg/registry/credentials"
+	runtime2 "cellery.io/cellery/components/cli/pkg/runtime"
```

Comment:
Can u rename this to something meaningful like cliruntime?

Reply:
Updated.